ev3dev2/display.py

```diff
@@ -375,14 +375,19 @@ def text_pixels(self, text, clear_screen=True, x=0, y=0, text_color='black', fon
 
         'font' : can be any font displayed here
             http://ev3dev-lang.readthedocs.io/projects/python-ev3dev/en/ev3dev-stretch/other.html#bitmap-fonts
+        - If font is a string, it is the name of a font to be loaded.
+        - If font is a Font object, returned from fonts.load(), then it is
```

Comment:
Can you make this a docs link? Something like this, I think:

    :meth:`ev3dev2.fonts.load`

Reply:
I added this reference link, and fixed the links to fonts, since they are not longer in the other.html, but in display.html.